### ISSUE TITLE
[V4] Dynamiczone error in GraphQL because of missing params

### ISSUE THREAD

**Issue by spathon** [strapi/strapi#11544]
### Describe the bug

Can't query dynamic zones in GraphQL

### Steps to reproduce the behavior

1.  Create a new Collection type with dynamic zones
2. Create a simple component
3. Try to query the collection and dynamic component in GraphQL

### Expected behavior
List the dynamic zones

### Screenshots

<img width="716" alt="Screenshot 2021-11-10 at 15 52 17" src="https://user-images.githubusercontent.com/454427/141137599-b2d3c25f-3d15-4c21-b434-e8365ee13b50.png">
<img width="1173" alt="Screenshot 2021-11-10 at 15 53 58" src="https://user-images.githubusercontent.com/454427/141137605-418984c9-b64c-401a-b029-8ff74fe87443.png">

### Code snippets

```graphql
query {
  testpages {
    data {
      id
      attributes {
        title
        dynContent {
          __typename
          ...on ComponentDemoCatDemoComp {
            id
            sometText
          }
        }
      }
    }
  }
}
```

### System

- Node.js version:  14.17.4
- NPM version: 6.14.14
- Strapi version: 4.0.0-beta.12
- Database: Mysql 5.7.28
- Operating system: MacOS Big Sur

**Comment by derrickmehaffy** [comment]
Confirmed on beta.12

**Comment by Convly** [comment]
Hello :wave:, this has been fixed in ##11553 and should be available in the next beta
